mne/preprocessing/ica.py

```diff
@@ -2079,3 +2079,239 @@ def _band_pass_filter(ica, sources, target, l_freq, h_freq, verbose=None):
         raise ValueError('Must specify both pass bands')
 
     return sources, target
+
+
+@verbose
+def corrmap(icas, template, threshold="auto", name="bads",
+            plot=True, inplace=False, ch_type="eeg"):
+    """Corrmap (Viola et al. 2009 Clin Neurophysiol) identifies the best group
+    match to a supplied template. Typically, feed it a list of fitted ICAs and
```

Comment:
For consistency, put a summary here that fits in the first line, mov detaile description 1 line down and add a blank line in between